spec/rails_helper.rb

```diff
@@ -53,6 +24,7 @@ def stub_omniauth
   
   config.before(:suite) do
     DatabaseCleaner.clean_with(:truncation)
+    SimpleCov.start 'rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.